src/server/frontend_xwayland/xwayland_surface.cpp

```diff
@@ -72,6 +72,32 @@ enum class NetWmMoveresize: uint32_t
     CANCEL = 11,        /* cancel operation */
 };
 
+// Any standard for the motif hints seems to be lost to time, but Weston has a reasonable definition:
+// https://github.com/wayland-project/weston/blob/f7f8f5f1a87dd697ad6de74a885493bcca920cde/xwayland/window-manager.c#L78
+namespace MotifWmHintsIndices
+{
+enum MotifWmHintsIndices: unsigned
```

Comment:
Not using enum class because the whole point of this enum is to index into a vector, for which I need explicit int conversion. Static casts would add a lot more code noise and yield no significant safety benefit.